src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,22 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+_INTERNAL_CALLERROR_MESSAGE = (
```

Comment:
Underscore between `CALL` and `ERROR` (corresponding to and due to the way `cygrpc.CallError` is capitalized).